cluster/ci/config/prow/jobs/README.md

```diff
@@ -0,0 +1,130 @@
+ci-operator templates
+=====================
+
+
+E2E tests
+---------
+
+The preferred way to write e2e tests is using `ci-operator`.  See the
+documentation for details on how to download, build, and execute it:
+
+https://github.com/openshift/ci-operator.git
+
+Many projects already have a `ci-operator` configuration file, which takes care
+of most of the CI process: downloading the source, building binaries, building
+RPMs, creating images, executing unit tests, etc.  These can be built upon for
+e2e test, requiring little or no modification.
+
+
+# Provisioning a cluster
+
+Contrary to other types of tests, E2E tests usually require a cluster, not just
+a single container.  While there aren't yet native primitives in `ci-operator`
+for cluster provisioning, it provides one open-ended feature that can be
+leveraged to accomplish that: template steps.
+
+Template steps allow the creation of arbitrary objects (e.g. pods) in the
```

Comment:
I do not understand how "create objects in cluster where CI pipeline is executed" (this sentence imo assumes that the cluster already exists) leads to "provisioning a cluster" (which I undrestand as new cluster creation, just for testing). When using templates, do I create an entirely new cluster or some special sub-cluster?

I guess this is both my personal question and doc clarification suggestion :)

Reply:
There are two clusters involved here: the one where `ci-operator` (and prow)
run, and the one provisioned exclusively to execute the tests.  I updated the
paragraph.